src/getting-started/toolchain.md

```diff
@@ -1,4 +1,10 @@
-## Rust Toolchain
+## Toolchain Installation
+
+<!---
+TODO: This needs some rework based on https://github.com/esp-rs/espup/issues/498
```

Comment:
It's really annoying that build-std isn't stable. If it were, I'd recommend that over any "rustup toolchain add" bits. Regardless of  the outcome of that issue, we still need to document installing esp toolchain for xtensa chips. I would assume riscv as the happy path, and then follow on to mention xtensa for the esp32,esp32s2 and esp32s3